bin/site

```diff
@@ -448,6 +448,38 @@
           (error! "Server response: " status " " (str/trim body))
           (message! "Request id: " (get headers "site-request-id")))))))
 
+(defn post-rule
```

Comment:
Could you comment as to the justification here? Currently all rules have to have names, we could relax that constraint but there are trade-offs. What's your opinion here?

Reply:
I started by looking at the openAPI spec. There's a POST verb on /rules/ https://github.com/juxt/site/blob/master/src/juxt/site/alpha/openapi.edn#L268 with an accompanying `:post-fn` dispatch. As I was jumping some hoops to get things started, I saw this as the shortest path to a working dev environment.

I have no particular preference as I don't know how apps will use it. I believe both method are valid. Though for the CLI I would use PUT as we know the resource ID beforehand and thus can easily DELETE / GET / PUT again.

Reply:
Ah right. I think that POST was left in by mistake after I switched to PUT. The whole rules insertion is questionable now we have 'post-resources' in bin/site. The benefit of PUT is that you get to specify the name, otherwise site has to come up with something - that something is unknowable and can complicate understanding and writing docs.

Reply:
reverted and changed the login-form deployment to use `post-resources` instead